lib/srfi/%3a176/version.sls

```diff
@@ -0,0 +1,18 @@
+(library (srfi :176 version)
+         (export version-alist)
+         (import (rnrs)
+                 (only (mosh config) get-config)
+                 (only (mosh) library-path))
+
+(define (version-alist)
+  `((version ,(get-config "version"))
+      (command "mosh")
```

Comment:
@okuoku 
I want to have "nmosh" if user is running nmosh.
Is there any way to know if user is running mosh or nmosh?

Reply:
Currently we have no direct way to do this so need to add some internal variable here to check https://github.com/higepon/mosh/blob/c4460bef6829262a163118b810f64528776eee3a/src/main.cpp#L296 . When mosh compiled as `nmosh`, C macro `WITH_NMOSH_DEFAULTS` will be defined.

Reply:
> . When mosh compiled as `nmosh`, C macro `WITH_NMOSH_DEFAULTS` will be defined.

Great. Thanks for the pointer. I was able to use the C macro.
CI is running now.